python/py-sherpa/Portfile

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+PortGroup           compilers 1.0
+name                py-sherpa
+version             1.2
```

Comment:
This version number contradicts the `github.setup` line below, which will overwrite it anyway. Move `github.setup` to the top.